NOTICE.txt

```diff
@@ -1,5 +1,5 @@
 Copyright (c) "Neo4j"
-Neo4j Sweden AB [http://neo4j.com]
+Neo4j Sweden AB [https://neo4j.com]
 
 This file is part of Neo4j.
```

Comment:
You might forgot to remove this phrase from this header.

Reply:
This looks as expected to me. It matches the `licensing/notice-apache2-prefix.txt` header supplied by the `org.neo4j.build:build-resources`.
Config: https://github.com/neo4j/neo4j-java-driver/pull/1496/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R507